src/app/db/tasks.py

```diff
@@ -0,0 +1,22 @@
+from databases import Database
+from fastapi import FastAPI
+
+from app.core.logging import logger
+from app.db.core import database_url
+
+
+async def connect_to_db(app: FastAPI) -> None:
+    database = Database(database_url(), min_size=2, max_size=10)
+
+    try:
+        await database.connect()
+        app.state._db = database
+    except Exception as e:
+        logger.warning(e)
```

Comment:
Using `logger.exception`, allows you to omit the exception argument (It will automagically add the exception including the stack trace to the generated log message)

Reply:
Done.